### DISCUSSION TITLE
why access  "138e7b1469f64156810a.jpg"

### DISCUSSION THREAD

**Discussion by joserand** [Kitware/trame#471] [General]
After I successfully visualized the model, I output the html file via the export_html function.
I wanted to show html files in bulk, so I used Flask for visualization. Here's the code:
    from pathlib import Path
    from flask import Flask, render_template, request
    from gevent import pywsgi
    
    app = Flask(__name__,
                template_folder='templates',)
    
    
    @app.route('/',methods=["POST","GET"])
    def index():
        print(request)
        index = request.values["html_address"]
        return render_template(index)
    
    # @app.route("/138e7b1469f64156810a.jpg",methods=["POST","GET"])
    # def index_picture():
    #     return render_template('index.html')
    
    if __name__ =="__main__":
        server = pywsgi.WSGIServer(('0.0.0.0',5000),app)
        server.serve_forever()

But I don't know what's wrong, every time I access an html file, I always access an image in passing. I see that the name of this image comes with the HTML, but since I am not proficient in HTML, I can't understand why this image is accessed.
![image](https://github.com/Kitware/trame/assets/89247520/b81a399f-8f4f-4187-9c35-050023f7aaa2)

Do you know why this image was accessed? Why do I sometimes see an access image before accessing a file, and sometimes after accessing a file? How can I avoid accessing this image? Thank you to everyone who answered.

**Comment by jourdain** [comment]
Not trame related. The issue comes from the html viewer that you are using.

**Comment by joserand** [comment]
Thank you for your answer.
